packages/docusaurus-1.x/package.json

```diff
@@ -35,15 +35,16 @@
     "@babel/register": "^7.0.0",
     "@babel/traverse": "^7.0.0",
     "@babel/types": "^7.1.2",
-    "autoprefixer": "^9.1.5",
+    "autoprefixer": "^9.6.0",
     "babylon": "^6.17.4",
     "chalk": "^2.4.2",
+    "chokidar": "^3.0.1",
```

Comment:
I wonder why this is added

Reply:
Probably for a security vulnerability fix. Could try removing it.